src/vs/workbench/parts/files/common/workingFilesModel.ts

```diff
@@ -333,6 +346,27 @@ export class WorkingFilesModel implements IWorkingFilesModel {
 		return this.mapEntryToResource[resource.toString()];
 	}
 
+	private recordRecentlyClosedEntries(resources: WorkingFileEntry[]): void {
+		if (resources.length === 0) {
+			return;
+		}
+
+		// Make the active entry the first entry
```

Comment:
See comment above, this is to ensure when restoring multiple files that the previously active file is opened in the editor.